src/commands/codepush/lib/file-utils.ts

```diff
@@ -27,6 +28,30 @@ export function copyFileToTmpDir(filePath: string): string {
   }
 }
 
+export async function moveReleaseFilesInTmpFolder(updateContentsPath: string): Promise<string> {
+    let tmpUpdateContentsPath: string = temp.mkdirSync("code-push");
+    tmpUpdateContentsPath = path.join(tmpUpdateContentsPath, "CodePush");
+    fs.mkdirSync(tmpUpdateContentsPath);
+
+    if (isDirectory(updateContentsPath)) {
+      await pfs.cp(normalizePath(updateContentsPath), normalizePath(tmpUpdateContentsPath));
+    } else {
+      const targetFileName = path.parse(updateContentsPath).base;
+      await pfs.cpFile(normalizePath(updateContentsPath), path.join(tmpUpdateContentsPath, targetFileName));
+    }
+
+    return Promise.resolve(tmpUpdateContentsPath);
```

Comment:
nit: no need to return `Promise` from the `async` function, should be wrapped automatically in a promise.

Reply:
@max-mironov Yeah, That's right. Fixed. Thanks!